pyomo/contrib/mindtpy/cut_generation.py

```diff
@@ -146,6 +170,19 @@ def add_oa_cuts(target_model, dual_values, solve_data, config,
 
 
 def add_int_cut(var_values, solve_data, config, feasible=False):
+    """
+    Modifies the model by adding integer cuts (if needed)
+
+    Parameters
+    ----------
+    var_values: TODO: Fill this in
```

Comment:
Values of the current variables to generate the cut. The input is generated as `var_values = list(v.value for v in fixed_nlp.MindtPy_utils.variable_list)` so its type is list.